pages/osx/md5.md

```diff
@@ -2,12 +2,19 @@
 
 > Calculate MD5 cryptographic checksums
 
-- Calculate the MD5 checksum for file(s) or files in a directory, one checksum per file
+- Print MD5 checksum for file
```

Comment:
I'd suggest "Calculate the MD5 checksums", especially since you're adding an example that isn't a file.